src/m365/teams/commands/team/team-app-list.ts

```diff
@@ -0,0 +1,119 @@
+import { Logger } from '../../../../cli/Logger';
+import GlobalOptions from '../../../../GlobalOptions';
+import { validation } from '../../../../utils/validation';
+import { aadGroup } from '../../../../utils/aadGroup';
+import GraphCommand from '../../../base/GraphCommand';
+import commands from '../../commands';
+import { formatting } from '../../../../utils/formatting';
+import { odata } from '../../../../utils/odata';
+
+interface CommandArgs {
+  options: Options;
+}
+
+interface Options extends GlobalOptions {
+  id?: string;
+  name?: string;
+}
+
+class TeamsTeamAppListCommand extends GraphCommand {
+  public get name(): string {
+    return commands.TEAM_APP_LIST;
+  }
+
+  public get description(): string {
+    return 'List apps installed in the specified team';
+  }
+
+  public defaultProperties(): string[] | undefined {
+    return ['id', 'displayName', 'distributionMethod'];
+  }
+
+  constructor() {
+    super();
+
+    this.#initTelemetry();
+    this.#initOptions();
+    this.#initValidators();
+    this.#initOptionSets();
+  }
+
+  #initTelemetry(): void {
+    this.telemetry.push((args: CommandArgs) => {
+      Object.assign(this.telemetryProperties, {
+        id: typeof args.options.id !== 'undefined',
+        name: typeof args.options.name !== 'undefined'
+      });
+    });
+  }
+
+  #initOptions(): void {
+    this.options.unshift(
+      {
+        option: '-i, --id [id]'
+      },
+      {
+        option: '-n, --name [name]'
+      }
```

Comment:
It's pretty late and should have been included in the specs, but actually the options should be `teamName` and `teamId`. Could you update this ... everywhere? 😓